modules/ravestate_nlp/__init__.py

```diff
@@ -40,7 +40,7 @@
                 "nlp:yesno"
         ))
     def nlp_preprocess(ctx):
-        text = ctx["rawio:in"]
+        text = str(ctx["rawio:in"]).lower()
```

Comment:
Consider calling all of internal processing after you made sure that you don't call conversion and lowercase on None. Is just a matter of taste however...

Reply:
good catch